src/main/scala/com/databricks/spark/xml/XmlInputFormat.scala

```diff
@@ -25,14 +25,17 @@ import org.apache.hadoop.io.{DataOutputBuffer, LongWritable, Text}
 import org.apache.hadoop.mapreduce.{InputSplit, RecordReader, TaskAttemptContext}
 import org.apache.hadoop.mapreduce.lib.input.{FileSplit, TextInputFormat}
 
+import scala.collection.mutable.ArrayBuffer
+
 /**
- * Reads records that are delimited by a specific start/end tag.
- */
+  * Reads records that are delimited by a specific start/end tag.
+  */
 class XmlInputFormat extends TextInputFormat {
 
   override def createRecordReader(
-      split: InputSplit,
-      context: TaskAttemptContext): RecordReader[LongWritable, Text] = {
+                                   split: InputSplit,
```

Comment:
Likewise this needs to be reverted

Reply:
I can't revert this change, because goal `scalacheckstyle` will fail on it.

![image](https://user-images.githubusercontent.com/5636129/50206193-b0961f80-037b-11e9-8296-4c6f163822a0.png)